src/User.js

```diff
@@ -9,6 +9,11 @@ class User {
         this.strideLength = data.strideLength;
         this.dailyStepGoal = data.dailyStepGoal;
         this.friends = data.friends;
+        this.firstName = '';
+    }
+    getFirstName() {
+        const [first, last] = this.name.split(' ');
```

Comment:
I really like your thinking here, I did of assigning the first and last name as parameters.  We can also shorten this if you would like.